MeshLib/CoordinateSystem.cpp

```diff
@@ -31,4 +30,23 @@ CoordinateSystem::CoordinateSystem(const Element &ele)
     }
 }
 
+unsigned char CoordinateSystem::getCoordinateSystem(const GeoLib::AABB &bbox) const
+{
+    unsigned char coords = 0;
+
+    const MathLib::Vector3 pt_diff(bbox.getMinPoint(), bbox.getMaxPoint());
+
+    // The axis aligned bounding box is a from the right half-open interval.
+    // Therefore, the difference between the particular coordinates of the
+    // points is modified by the unit in the last place towards zero.
+    if (std::nexttoward(std::abs(pt_diff[0]), 0.0) > .0)
```

Comment:
Yes, this can happen, but at this point we have not enough information to handle this.